### PR TITLE
Grant/research guide

### PR DESCRIPTION
**Pull request by grantgoodmanedX**
## [DOC-3750](https://openedx.atlassian.net/browse/DOC-3750)

I've made some changes to restructure the Research Guide. I have removed most of the front matter (except for the bit at the bottom of the index.html page) and added a Getting Started chapter. I also pulled most of the information from the Data Czar chapter into a new Using the Data Package chapter.

In the future, I'd like to add some specific use cases, as called for in DOC-3656 (Add an FAQ section).

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Subject matter expert: @stroilova 
- [x] Doc team review (dev edit): @edx/doc

FYI: Tag anyone else who might be interested in this PR here.

### Testing

- [x] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

- [x] RTD draft at http://draft-research-guide-sep17.readthedocs.io/en/latest/index.html

### Post-review

- [ ] Squash commits

### CODE REVIEW

File: en_us/data/source/access/credentials.rst [hunk 1]
```diff
@@ -34,15 +35,15 @@ download data packages on an ongoing basis.
 Data Czar: Create Keys for Encryption and Decryption
 ****************************************************************
 
-To assure the security of data packages, the edX Analytics team encrypts all
-files before making them available to a partner institution. As a result, when
-you receive a data package (or other files) from the edX Analytics team, you
-must decrypt the files that it contains before you use them.
+To assure the security of data packages, edX encrypts all files before making
```
Comment (catong) [inline en_us/data/source/access/credentials.rst]: assure -> ensure

File: en_us/data/source/access/credentials.rst [hunk 2]
```diff
@@ -149,17 +147,17 @@ EdX: Deliver Credentials for Accessing Data Storage
...
+to the Amazon Web Services (AWS) Simple Storage Service (Amazon S3). EdX
+creates an individual account to access this storage service for
 each data czar. The credentials for accessing this account are called an Access
 Key and a Secret Key.
 
-After the edX Analytics team creates these access credentials for you, they use
-the public encryption key that you sent your partner manager to encrypt the
-credentials into a **credentials.csv.gpg** file. The edX Analytics team then
-sends the file to you as an email attachment.
+After edX creates these access credentials for you, they use the public
```
Comment (catong) [inline en_us/data/source/access/credentials.rst]: "they" (in "...they use the public encryption key") no longer agrees with "edX" vs "the edX Analytics team"

File: en_us/data/source/data_czars/index.rst [hunk 3]
```diff
@@ -1,23 +1,154 @@
...
 
 A data czar is the single representative at a partner institution who has the
 credentials to download and decrypt edX data packages. The data czar is
 responsible for transferring data securely to researchers and other interested
 parties after it is received. Due to the sensitivity of this data, the
 responsibility for these activities is restricted to one individual.
 
-At each partner institution, the data czar is the primary point of contact for
-information about edX data.
+A data czar is the single representative at a partner institution who has the
```
Comment (catong) [inline en_us/data/source/data_czars/index.rst]: The first sentence in this and the preceding paragraph are the same.

**Comment by catong** [comment]
@grantgoodmanedX This looks great; it's a big improvement. I made some minor notes in the comments. I saw one existing heading, now top level in the TOC, that reads quite awkwardly, and I wonder if you can come up with something better: Data Czar and Team Selection and Responsibilities. Maybe just "Responsibiities of the Data Czar and Team"?

**Review by catong — APPROVED** [review]
Looks great; some edits in comments

**Comment by grantgoodmanedX** [comment]
Thanks, @catong , I've pushed edits to address your comments.

**Comment by grantgoodmanedX** [comment]
Replaced by https://github.com/edx/edx-documentation/pull/1602

**Pull request closed without merging**